app/src/main/res/layout-sw600dp/activity_settings.xml

```diff
@@ -0,0 +1,155 @@
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/settings_margin_default"
+            android:text="@string/settings_head_tracking"
+            android:textColor="@color/textColor"
+            android:textSize="@dimen/settings_label_text_size"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/head_tracking_switch" />
+
+        <androidx.appcompat.widget.SwitchCompat
+            android:id="@+id/head_tracking_switch"
+            android:layout_width="@dimen/settings_switch_width"
+            android:layout_height="@dimen/settings_switch_height"
+            android:layout_marginTop="@dimen/settings_margin_default"
+            android:layout_marginEnd="@dimen/settings_margin_default"
+            android:layout_marginBottom="@dimen/settings_margin_default"
+            android:checked="true"
+            android:scaleX="1.5"
+            android:scaleY="1.5"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+    </com.willowtree.vocable.customviews.VocableConstraintLayout>
+
+    <com.willowtree.vocable.customviews.ActionButton
+        android:id="@+id/privacy_policy_button"
+        style="@style/ActionButton"
+        android:layout_marginStart="@dimen/settings_action_button_margin"
+        android:layout_marginEnd="@dimen/settings_margin_small"
+        android:layout_marginBottom="@dimen/settings_action_button_bottom_margin"
+        android:text="@string/privacy_policy"
+        app:layout_constraintBottom_toTopOf="@+id/version"
+        app:layout_constraintEnd_toStartOf="@+id/contact_devs_button"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <com.willowtree.vocable.customviews.ActionButton
+        android:id="@+id/contact_devs_button"
+        style="@style/ActionButton"
+        android:layout_marginEnd="@dimen/settings_action_button_margin"
+        android:layout_marginBottom="@dimen/settings_action_button_bottom_margin"
+        android:text="@string/contact_developers"
+        app:layout_constraintBottom_toTopOf="@+id/version"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/privacy_policy_button" />
+
+    <TextView
+        android:id="@+id/version"
+        style="@style/VersionTextView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/settings_bottom_margin"
+        android:text="@string/version"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintVertical_chainStyle="packed" />
+
+    <include
```

Comment:
Don't forget to update the dimens for the confirmation screen

Reply:
I looked at this screen in the emulator, and it looks okay. I don't think there's anything I need to change, but let me know if there is

Reply:
<img width="810" alt="Screen Shot 2020-03-03 at 3 53 26 PM" src="https://user-images.githubusercontent.com/25698573/75818706-5f4b3380-5d67-11ea-9afd-9752b597b6a3.png">